bin/runcutest

```diff
@@ -467,7 +470,7 @@ if [[ ${PKG} == "matlab" ]]; then
         error "${thisprog}: environment variable MYMATLAB not set"
         exit 10
     fi
-    MEXFORTRAN=${MYMATLAB}/bin/mex
+    MEXFORTRAN=${MYMATLAB}/bin/MEX
```

Comment:
Why has this changed from mex to MEX? There is no binary MEX with my Linux Matlab, only mex. Is this an issue with Matlab on non-Linux systems?

Reply:
Oh, I think I forgot to remove this.